### PR TITLE
Tzachi sensor sn 2

### PR DESCRIPTION
**Pull request by tzachi-dar**
This code calculates the libre serial number and also sends it to the OOP algorithm.

It replaces https://github.com/NightscoutFoundation/xDrip/pull/445.
It will store both tomato and blukon on the same location.

### CODE REVIEW

File: wear/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/Blukon.java [hunk 1]
```diff
@@ -52,7 +52,7 @@
         INITIAL
     }
 
-    private static final boolean testWithDeadSensor = true; // never in production
+    private static final boolean testWithDeadSensor = false; // never in production
```
Comment (dabear) [inline wear/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/Blukon.java]: Is this change intentional?

Comment (tzachi-dar) [reply wear/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/Blukon.java]: This is a bugfix, actually already fixed in https://github.com/NightscoutFoundation/xDrip/pull/447

**Comment by dabear** [comment]
Looks good to me. Sensor serial number is correct (verified against Abbot's libre reader handheld)
image

**Comment by tzachi-dar** [comment]
Ready for merge.
